enterprise_learner_portal/api/v1/serializers.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+enterprise_learner_portal serializer
+"""
+from __future__ import absolute_import, unicode_literals
+
+from rest_framework import serializers
+
+try:
+    from lms.djangoapps.certificates.api import get_certificate_for_user
+    from lms.djangoapps.program_enrollments.api.api import (
+        get_due_dates,
+        get_course_run_url,
+        get_emails_enabled,
```

Comment:
How many of these helper functions make DB calls or external API calls? I'm a bit worried about performance, but perhaps those concerns are premature at this point.

Reply:
Some, but not all. https://github.com/edx/edx-platform/pull/21258/files in api/api.py. get_due_dates looks to be one that does